code/modules/projectiles/ammo_datums.dm

```diff
@@ -357,9 +363,6 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 	penetration = 5
 	accuracy = -15
 
-/datum/ammo/bullet/revolver/heavy/on_hit_mob(mob/M,obj/projectile/P)
-	staggerstun(M, P, stagger = 1, slowdown = 0.5, knockback = 1)
```

Comment:
Another one?

Reply:
Nope.

Is just a duplicate of the parent type effects, probably added before regular revolver rounds got it and overlooked.